arquillian-chameleon-deployments/arquillian-chameleon-deployment-api/src/main/java/org/arquillian/container/chameleon/deployment/api/ConstantException.java

```diff
@@ -0,0 +1,4 @@
+package org.arquillian.container.chameleon.deployment.api;
+
+public class ConstantException extends Exception {
```

Comment:
What is the need for this class? Could be good to add Javadoc explaining its existance. Maybe also name could be changed to suggest what is it really used for?

Reply:
Because an enum cannot return `null`

Reply:
I understand that, but I am a bit unhappy with the name of this exception as it doesn't really explain its reason to exists. It's also publicly available - should it be?